src/bmark/run_solver.jl

```diff
@@ -34,31 +34,46 @@ function solve_problems(solver :: Function, problems :: Any;
   nprobs = length(problems)
   solverstr = split(string(solver), ".")[end]
 
-  counter_fields = collect(fieldnames(Counters))
-  ncounters = length(counter_fields)
-  types = [Int; String;   Int;   Int;  Symbol;    Float64;       Float64;   Int;    Float64; fill(Int, ncounters); String]
-  names = [:id;  :name; :nvar; :ncon; :status; :objective; :elapsed_time; :iter; :dual_feas;                    counter_fields; :extrainfo]
+  f_counters = collect(fieldnames(Counters))
+  fnls_counters = collect(fieldnames(NLSCounters))[2:end] # Excludes :counters
+  ncounters = length(f_counters) + length(fnls_counters)
+  types = [Int; String;   Int;   Int;   Int;  Symbol;    Float64;       Float64;   Int;    Float64; fill(Int, ncounters); String]
+  names = [:id;  :name; :nvar; :ncon; :nequ; :status; :objective; :elapsed_time; :iter; :dual_feas; f_counters; fnls_counters; :extrainfo]
```

Comment:
Should `types` and `names` be global constants (with better names)?

Reply:
I don't think we're needing them in other places, but we can access them with `eltypes(df)` and `names(df)`.